pgraph/graphsync_test.go

```diff
@@ -92,3 +92,40 @@ func TestGraphSync2(t *testing.T) {
 		t.Errorf("%s", s)
 	}
 }
+
+func TestGraphSync3(t *testing.T) {
+	v1 := NV("v1")
+	v2 := NV("v2")
+	v3 := NV("v3")
+
+	e1 := NE("e1")
+	e2 := NE("e2")
+	e3 := NE("e3")
+	e4 := NE("e4")
+
+	// g base graph with 3 edges.
```

Comment:
Only full sentences need punctuation. Please remove the trailing period here, and on the comment below.